src/segment_language.go

```diff
@@ -98,6 +99,8 @@ const (
 	EnableVersionMismatch Property = "enable_version_mismatch"
 	// HomeEnabled displays the segment in the HOME folder or not
 	HomeEnabled Property = "home_enabled"
+	// LanguageExtensions the list of extensions to validate
+	LanguageExtensions Property = "extensions"
```

Comment:
silent launch? Let's see if it works according to expectations :-)

Reply:
Power user only 😅